plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -142,6 +142,7 @@ def keycloak_argument_spec():
         validate_certs=dict(type='bool', default=True),
         connection_timeout=dict(type='int', default=10),
         token=dict(type='str', no_log=True),
+        refresh_token=dict(type='str', no_log=True),
```

Comment:
You should add this to the doc_fragment as well ;-)

Reply:
Done; thank you for pointing out the correct place to add it.

I marked it as added in 10.2.0, since I see that is the next expected version, and I'll update it if that is released before this is merged. I suggest we leave this conversation unresolved until checking that before merging.